### DISCUSSION TITLE
Dreambooth Google Colab error after new update

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#2619] [Q&A]
I'm getting this error on this New update, already tried to copy/paste new dependecies and model download but I'm still getting this error. Any suggestion? I like to use an old dreambooth notebook.

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/121369033/f75c7f4a-a555-4b66-918f-bc69ad65f66c)

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/121369033/a9695106-35c9-4573-abab-a7e17b8b68e1)

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/121369033/373242fc-9675-41ed-a230-3b17e1c407b8)

**Comment by cutecoelho** [comment]
Same problem here! I was needing to use a lot today but I'm stuck on this error :(

**Reply by i-huzaifa-arshad** [reply]
same problem. Hope it get resolved soon

**Comment by TheLastBen** [comment]
fixed

**Reply by meunumerotim333** [reply]
thank u ♥

**Comment by meunumerotim333** [comment]
Well, the problem is fixed but appeared another one. I've tried train 2 times and is showing the same error on A1111

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/121369033/5f7b011b-7bbc-406d-a480-f194f29a0966)

is saying the CKPT could be malicious, and is the same file I've trained more than 200 models.. is the first time that shows up after this update.

**Reply by TheLastBen** [reply]
That's not the colab

**Reply by meunumerotim333** [reply]
I know, but isn't the Colab generating malicious file now? Because I did the same process I always do.

**Reply by TheLastBen** [reply]
it's not generating malicious files, add `--disable-safe-unpickle` to your A1111
